### PR TITLE
CR-9099 editable ingress host

### PR DESCRIPTION
**Pull request by rotem-codefresh**

### CODE REVIEW

**Review by noam-codefresh — APPROVED** [review]

File: cmd/commands/common.go [hunk 1]
```diff
@@ -427,27 +444,45 @@ func getIngressHostFromCluster(ctx context.Context, opts *RuntimeInstallOptions)
...
 			} else {
-				opts.IngressHost = fmt.Sprintf("https://%s", ingress.IP)
+				foundIngressHost = fmt.Sprintf("https://%s", ingress.IP)
 			}
 			break
 		}
 	}
 
+	if store.Get().Silent {
+		log.G(ctx).Warn("Using ingress host %s", foundIngressHost)
```
Comment (roi-codefresh) [inline cmd/commands/common.go]: Warnf**

Comment (roi-codefresh) [reply cmd/commands/common.go]: `Warn` is without interpolation `Warnf` is with.

File: cmd/commands/common.go [hunk 2]
```diff
@@ -427,27 +444,45 @@ func getIngressHostFromCluster(ctx context.Context, opts *RuntimeInstallOptions)
...
 
+	var foundIngressHost string
+
 	for _, s := range ServicesList.Items {
-		if s.ObjectMeta.Name == opts.IngressController {
+		if s.ObjectMeta.Name == opts.IngressController && s.Spec.Type == "LoadBalancer" {
 			ingress := s.Status.LoadBalancer.Ingress[0]
 			if ingress.Hostname != "" {
-				opts.IngressHost = fmt.Sprintf("https://%s", ingress.Hostname)
+				foundIngressHost = fmt.Sprintf("https://%s", ingress.Hostname)
```
Comment (roi-codefresh) [inline cmd/commands/common.go:455]: why don't you break out of here if you found a result?

**Pull request merged**